File: specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/examples-js/RuleSets_Create.js
const { CdnManagementClient } = require("@azure/arm-cdn");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Creates a new rule set within the specified profile.
 *
 * @summary Creates a new rule set within the specified profile.
 * x-ms-original-file: specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/examples/RuleSets_Create.json
 */
async function ruleSetsCreate() {
  const subscriptionId = "subid";
  const resourceGroupName = "RG";
  const profileName = "profile1";
  const ruleSetName = "ruleSet1";
  const credential = new DefaultAzureCredential();
  const client = new CdnManagementClient(credential, subscriptionId);
  const result = await client.ruleSets.create(resourceGroupName, profileName, ruleSetName);
  console.log(result);
}

ruleSetsCreate().catch(console.error);

File: specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/examples-java/RuleSets_Create.java
/** Samples for RuleSets Create. */
public final class Main {
    /*
     * x-ms-original-file: specification/cdn/resource-manager/Microsoft.Cdn/stable/2023-05-01/examples/RuleSets_Create.json
     */
    /**
     * Sample code: RuleSets_Create.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void ruleSetsCreate(com.azure.resourcemanager.AzureResourceManager azure) {
        azure
            .cdnProfiles()
            .manager()
            .serviceClient()
            .getRuleSets()
            .createWithResponse("RG", "profile1", "ruleSet1", com.azure.core.util.Context.NONE);
    }
}
